### PR TITLE
Consolidate configuration into one place.

### PR DESCRIPTION
**Pull request by ghost**
This greatly expands the functionality of the templates used for our deployments. The end result is the same, but we have a lot more power over how we get there.

Demo: soon™

Other relevant PRs:
https://github.com/cloudigrade/cloudigrade/pull/327
https://github.com/cloudigrade/frontigrade/pull/19

### CODE REVIEW

**Review by infinitewarp — DISMISSED** [review]
That's a lot of words. Didn't see anything freaky along the way. I'll trust it works. 🙂

File: README.rst [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+If you'd like to start the cluster, and deploy Cloudigrade along with supporting services run the following:
+
+.. code-block:: bash
+
+    # When deploying cloudigrade make sure you have AWS_ACCESS_KEY_ID and
+    # AWS_SECRET_ACCESS_KEY set in your environment or the deployment will fail
+    make oc-up-all
+
+This will create the **ImageStream** to track **PostgreSQL:9.6**, create the templates for **RabbitMQ** and **cloudigrade**, and finally use the templates to create all the objects necessary to deploy **cloudigrade** and the supporting services. There is a chance that the deployment for **cloudigrade** will fail due to the db not being ready before the mid-deployment hook pod is being run. Simply run the following command to trigger a redemployment for **cloudigrade**:
```
Comment (abaiken) [inline README.rst]: Should we remove that it creates the templates for RabbitMQ since we killed the rabbit?

File: ocp/local.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+        secretAccessKey: 456
+      django:
+        debug: true
+        allowedHosts: "*"
+        settingsModule: config.settings.prod
+        secretKey: local
+      db:
+        secretsFileName: postgresql
+        databaseHost: postgresql.myproject.svc
+      rabbitmq:
```
Comment (abaiken) [inline ocp/local.yaml]: Same kind of comment as above (should we remove this since rabbit is dead?).

File: ocp/test.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+        secretAccessKey:
+      django:
+        debug: true
+        allowedHosts: '*'
+        settingsModule: config.settings.prod
+        secretKey:
+      db:
+        secretsFileName: postgresql
+        databaseHost: postgresql.cloudigrade-test.svc
+      rabbitmq:
```
Comment (abaiken) [inline ocp/test.yaml]: same comment as above

**Review by abaiken — CHANGES_REQUESTED** [review]
I'm assuming we need to remove references to the 🐇 from this. Definitely correct me if I'm wrong!

File: README.rst (hunk 1 above)
Comment (infinitewarp) [reply README.rst]: Yep, good catch! 🐰🔫

**Comment by ghost** [comment]
@abaiken I could have sworn I purged all of 🐇 from the repo, but I guess I missed somethings, thanks for noticing!

**Comment by ghost** [comment]
@abaiken Changes made!

In addition to thoroughly 🔪 🐇 I removed the `AWS_SQS_URL` from templating as `cloudigrade` does an excellent job of building it, itself. I also added the missing env vars to travis so our build no longer fails 😅

**Review by abaiken — APPROVED** [review]

**Pull request merged**